### PR TITLE
WebClient: Fix bug JobVacancies Page 

### PR DESCRIPTION
**Pull request by Muhammadislom99**

### CODE REVIEW

**Review by Ubaydulloev1104 — APPROVED** [review]

File: src/MRA.Jobs.Api/Application/Features/VacancyCategories/Queries/GetJobCategoriesQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    public GetJobCategoriesQueryHandler(IApplicationDbContext context, IMapper mapper)
+    {
+        _context = context;
+        _mapper = mapper;
+    }
+
+    public async Task<List<JobCategoriesResponse>> Handle(GetJobCategoriesQuery request, CancellationToken cancellationToken)
+    {
+        var jobs = (await _context.JobVacancies.ToListAsync()).AsEnumerable();
```
Comment (Firuz2006) [inline src/MRA.Jobs.Api/Application/Features/VacancyCategories/Queries/GetJobCategoriesQueryHandler.cs:20]: You may remove ToListAsync.

var jobsQuery=_context.JobVacancies;

**Review by Firuz2006 — CHANGES_REQUESTED** [review]

File: src/MRA.Jobs.Api/Application/Features/VacancyCategories/Queries/GetJobCategoriesQueryHandler.cs (hunk 1 above)
Comment (Muhammadislom99) [reply src/MRA.Jobs.Api/Application/Features/VacancyCategories/Queries/GetJobCategoriesQueryHandler.cs:20]: I can’t delete, there is a condition, look at the code below, if there is an alternative solution, I’m waiting

Comment (Firuz2006) [reply src/MRA.Jobs.Api/Application/Features/VacancyCategories/Queries/GetJobCategoriesQueryHandler.cs:20]: You may call tolist after building the query. This way you immediately get grouped and filtered data from the database

Comment (Muhammadislom99) [reply src/MRA.Jobs.Api/Application/Features/VacancyCategories/Queries/GetJobCategoriesQueryHandler.cs:20]: how ?

Comment (Muhammadislom99) [reply src/MRA.Jobs.Api/Application/Features/VacancyCategories/Queries/GetJobCategoriesQueryHandler.cs:20]: send me sample code please.

**Review by adheeeem — APPROVED** [review]

**Review by Ubaydulloev1104 — APPROVED** [review]

**Review by Firuz2006 — APPROVED** [review]

**Review by Ubaydulloev1104 — APPROVED** [review]

**Pull request merged**